test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/RetrieveTransactionResourceTests.Instance.cs

```diff
@@ -37,7 +38,14 @@ public async Task GivenSupportedAcceptHeaders_WhenRetrieveInstance_ThenServerSho
 
             using DicomWebResponse<DicomFile> response = await _client.RetrieveInstanceAsync(instanceId.StudyInstanceUid, instanceId.SeriesInstanceUid, instanceId.SopInstanceUid, transferSyntax);
 
-            Assert.Equal(DicomWebConstants.ApplicationDicomMediaType, response.ContentHeaders.ContentType.MediaType);
+            Assert.Equal(DicomWebConstants.MultipartRelatedMediaType, response.ContentHeaders.ContentType.MediaType);
```

Comment:
This will be not needed if the client dll by default uses application/dicom.

Should we add another E2E test to test E2E multipart accept works?

Reply:
Yes you are right, this is not needed if the client uses older format. Sure, if the client uses single part by default then we need to have E2E for multipart